File-Formats/Presentation/Working-with-images.md

```diff
@@ -299,6 +299,95 @@ pptxDoc.Close()
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Images/Replace-SVG-Image).
 
+## Crop Image
+
+Refine the image dimensions by cropping out any unintended elements, using a cropping tool to focus the image on relevant areas.
```

Comment:
Crop the images in an existing presentation or insert the cropped picture while creating a presentation from scratch by applying crop properties using the .NET PowerPoint Library.